### PR TITLE
Skip converter namespace if no custom types

### PR DESCRIPTION
**Pull request by dmondrik**
### What does this Pull Request accomplish?

In some cases, we were generating an empty namespace at the bottom of the service header. This PR now avoids doing that.

### Why should this Pull Request be merged?

[Technical Debt 2192495](https://ni.visualstudio.com/DevCentral/_workitems/edit/2192495): [gRPC] Fix issue with empty namespace generating at bottom of _service.h

### What testing has been done?

Built locally. Verified that the only generated change was this one for FGEN.

### CODE REVIEW

File: examples/session/session-reservation.py [hunk 1]
```diff
@@ -80,7 +80,7 @@
         )
     )
     vi = init_with_options_response.vi
-    print(f"Session initialized with name {session_name} and id {vi.id}.\n")
+    print(f"Session initialized with name {session_name}.\n")
```
Comment (dmondrik) [inline examples/session/session-reservation.py:83]: I know why this change is needed, given Chris' recent submission. I have no idea how his PR got through without including this. FYI @astarche @reckenro

Comment (reckenro) [reply examples/session/session-reservation.py:83]: Agreed, this should've failed the `validate_python` step in Chris' PR or at least his merge to main. I created a Tech Debt [AB#2219712](https://dev.azure.com/ni/DevCentral/_workitems/edit/2219712) item to investigate and hopefully figure out / fix what's going on here.

**Review by reckenro — APPROVED** [review]
